### ISSUE TITLE
Identifiers in getter `MethodDefinition` nodes are collected as references

### ISSUE THREAD

**Issue by goto-bus-stop** [goto-bus-stop/scope-analyzer#3]
See https://github.com/goto-bus-stop/browser-pack-flat/issues/28

```js
var url = {}
class Request {
  get url () { return url.format() }
}
```

`get url ()` is detected as a reference to `url`.
